index.js

```diff
@@ -57,16 +57,14 @@ LessCompiler.prototype.build = function() {
       this.inputFile,
       this.inputPaths
     ),
-    paths: this.inputPaths.slice()
+    paths: []
   };
 
-  this.inputPaths = lessOptions.paths.slice();
```

Comment:
Also, I removed this line, as it seems pretty useless (reassigning the same (sliced twice) array again). I hope I am not missing something though?